gpkit/constraints/bounded.py

```diff
@@ -74,26 +79,34 @@ def __init__(self, constraints, substitutions=None, verbosity=1,
 
     def sens_from_dual(self, las, nus):
         "Return sensitivities while capturing the relevant lambdas"
-        self.bound_las = las[-2*len(self.bounded_varkeys):]
+        n = sum(map(bool, [self.lowerbound, self.upperbound]))
```

Comment:
I think this would be clearer as `n = bool(self.lowerbound) + bool(self.upperbound)`